deploy/registration-service.yaml

```
@@ -222,13 +223,13 @@ objects:
       namespace: ${NAMESPACE}
       labels:
         provider: codeready-toolchain
-        run: registration-service
+        run: proxy-metrics
     spec:
       ports:
         - name: "8082"
```

```suggestion
        - name: metrics
```
we can name the port on the service with `metrics`, too